fix JOB - Integrations and Incidents Health Check - Lists handling (#15010)
demisto/content: Packs/IntegrationsAndIncidentsHealthCheck/ReleaseNotes/1_2_6.md

#### Playbooks
##### JOB - Integrations and Incidents Health Check - Lists handling

> - Fixed an issue where *Set lists names to context* (task 54) returned output that included the names and contents of XSOARlists when only the names were needed.

Reviewer: ```suggestion
- Fixed an issue where *Set lists names to context* (task 54) returned both the names and contents of the XSOAR lists when only the names were needed.
```
